src/main/java/uk/gov/justice/hmpps/prison/PrisonApiServer.java

```diff
@@ -3,11 +3,10 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.domain.EntityScan;
-import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 import org.springframework.scheduling.annotation.EnableScheduling;
 
+
 @SpringBootApplication(scanBasePackages = { "uk.gov.justice.hmpps" })
-@EnableJpaRepositories(basePackages = { "uk.gov.justice.hmpps" })
```

Comment:
This has been pulled out into a separate @Configuration class to avoid JPA being loaded with the WebMvcTest tests